GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTPrioritizer.m

```diff
@@ -55,6 +57,11 @@ - (instancetype)init {
 #pragma mark - GDTCORPrioritizer Protocol
 
 - (void)prioritizeEvent:(GDTCOREvent *)event {
+  if (event.customPrioritizationParams[@"needs_network_connection_info"]) {
+    gdt_cct_NetworkConnectionInfo networkInfo = GDTCCTConstructNetWorkConnectionInfo();
+    NSData *networkInfoData = [NSData dataWithBytes:&networkInfo length:sizeof(networkInfo)];
```

Comment:
This works only because gdt_cct_NetworkConnectionInfo has a static length. My recommendation to change the return value of GDTCCTConstructNetWorkConnectionInfo to NSData is to help protect usage of the API in particular. This could be a harder bug to track down in the future if the NSData conversion occurs here.